packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -171,6 +209,22 @@ class NetworkController
       currentValues.add(webSocket);
     }
 
+    if (timeline != null) {
+      _processTimeline(
```

Comment:
in what case will `timeline != null`? is this for backwards compatibility?

Reply:
Yes, this is for backward compatibility. When we start shipping a pinned DevTools with the SDK we can tear this all out.